test/SNU_NPB_SER_C/dc/Makefile

```diff
@@ -32,8 +34,11 @@ dump:
 	sudo $(CRIU) dump -vvv --shell-job -o dump.log -t $(PID)
 
 transform:
-	sudo $(CRIT) recode $(CURR) $(CURR)/$(TGT) $(TGT) $(BIN) $(BINDIR) $(DEBUG) 
+	sudo $(PYTHON) $(CRIT) recode $(CURR) $(CURR)/$(TGT) $(TGT) $(BIN) $(BINDIR) $(DEBUG) 
```

Comment:
Is this because of the code only working for python3 now? This can be done within crit, we don't need to specify python3 in the makefiles